### PR TITLE
Feat: Project Euler Problem 10

### PR DESCRIPTION
**Pull request by domix80**
Adding resolution for Euler Problem 10

### CODE REVIEW

**Comment by domix80** [comment]
@aparibocci, Could you please review it?

**Comment by domix80** [comment]
@StepfenShawn - May I ask you to review and eventually merge the code?

File: project_euler/problem_010/sol1.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#Project Euler Problem 10: #https://projecteuler.net/problem=10
+#Summation of primes
+#The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+#Find the sum of all the primes below two million.
+#References: - https://en.wikipedia.org/wiki/Prime_number
```
Comment (StepfenShawn) [inline project_euler/problem_010/sol1.rb:5]:
```suggestion
#References:  https://en.wikipedia.org/wiki/Prime_number
```

File: project_euler/problem_010/sol1.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#Project Euler Problem 10: #https://projecteuler.net/problem=10
```
Comment (StepfenShawn) [inline project_euler/problem_010/sol1.rb:1]:
```suggestion
#Project Euler Problem 10:  https://projecteuler.net/problem=10
```

**Review by StepfenShawn — CHANGES_REQUESTED** [review]
A few changes about comments.

**Pull request merged**